tests/test_root_commands.py

```diff
@@ -69,6 +69,11 @@ def test_attr_only_second_argument_if_writable(hlwm):
         .returncode == 7
 
 
+def test_set_attr_can_not_set_writable(hlwm):
+    assert hlwm.call_xfail_no_output('set_attr tags.count 5') \
```

Comment:
I'm a bit late to the party, but why is it "no output", if it should emit the error message that got fixed in this PR?

And as a follow-up question: It looks like `call_xfail_no_output` doesn't even assert that there is literally no output. Shouldn't it do that?

Reply:
Right, I think this just came out of the tab completion. I've fixed this in #440.